package.json

```diff
@@ -25,6 +25,7 @@
     "test": "jest --runInBand",
     "int-test": "cypress run --config video=false",
     "int-test-ui": "cypress open",
+    "int-test-video": "CYPRESS_NO_COMMAND_LOG=1 cypress run --config video=true,slowTestThreshold=15000 --env COMMAND_DELAY=1200",
```

Comment:
Do we need the `slowTestThreshold`? Seems dissimilar to the rest of the changes

Reply:
Is this is to avoid the delay triggering slow test warnings?

Reply:
yeah, adding the delays causes all the tests to trigger the slow warnings, so resetting the threshold for these tests